hw/top_earlgrey/formal/top_earlgrey_fpv_ip_cfgs.hjson

```diff
@@ -19,33 +19,28 @@
   rel_path: "hw/top_earlgrey/formal/ip/summary"
 
   use_cfgs: [
+             // Use non-parameterized pinmux.
              {
                name: pinmux_fpv
                dut: pinmux_tb
                fusesoc_core: lowrisc:fpv:pinmux_fpv
                import_cfgs: ["{proj_root}/hw/formal/tools/dvsim/common_fpv_cfg.hjson"]
-               rel_path: "hw/ip/pinmux/{sub_flow}/{tool}"
+               rel_path: "hw/top_earlgrey/ip_autogen/pinmux/{sub_flow}/{tool}"
                defines: "FPV_ALERT_NO_SIGINT_ERR"
-               exp_fail_hjson: "{proj_root}/hw/ip/pinmux/fpv/pinmux_expected_failure.hjson"
+               exp_fail_hjson: "{proj_root}/hw/top_earlgrey/ip_autogen/pinmux/fpv/pinmux_expected_failure.hjson"
                cov: true
              }
 
              // Use chip_eargrey_asic parameters to verify pinmux.
              {
                name: pinmux_chip_fpv
                dut: pinmux_tb
```

Comment:
I think this line should rather be:
```
dut: pinmux_chip_tb
```
Because `pinmux_tb.sv` is not included in the pinmux_chip_fpv FuseSoC core.

Reply:
Certainly!